news/_posts/2020-04-27-Polymake.jl-0.4-has-been-tagged.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+title: Polymake.jl 0.4 has been tagged
+author: Marek Kaluba
+---
+
+`Polymake.jl` v0.4.0 has been tagged!
+
+Since we're upgrading to support `CxxWrap.jl` `v0.10.1`, we decided to bump the major 
+version to force compilation of libpolymake against libcxxwrap-julia. 
+The release has been somewhat hastened by the need for `Oscar.jl` jointly working with 
+the newest `CxxWrap.jl`, so expect `v.0.4.1` and later minor versions rather heavy
+in added features :)
```

Comment:
```suggestion
in added features :).
```

Reply:
I actually don't find the period great here, it interferes with the interpretation of `:)` as an emoji. I personally am fine with using an emoji as closing punctuation, but would otherwise find it clearer as "features. :)" (quite subjective I know ;-) )
cf. https://rfourquet.github.io/oscar-website/news/2020/04/27/Polymake.jl-0.4-has-been-tagged/